bin/snare

```diff
@@ -148,14 +146,15 @@ if __name__ == '__main__':
     parser.add_argument("--server-header", help="set server-header", default='nginx/1.3.8')
     parser.add_argument("--no-dorks", help="disable the use of dorks", type=str_to_bool,  default=True)
     parser.add_argument("--log-dir", help="path to directory of the log file", default='/opt/snare/')
+    parser.add_argument("--path", help="path to save the page to be cloned", required=False, default='/opt/')
```

Comment:
I suggest you reuse this path for both logs and snare files, what do you think?

Reply:
I'd same thing on my mind